COMET.Web.Common/Server/Services/NamingConventionService/NamingConventionService.cs

```diff
@@ -0,0 +1,68 @@
+// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="NamingConventionService.cs" company="RHEA System S.A.">
+//    Copyright (c) 2023 RHEA System S.A.
+// 
+//    Authors: Sam Gerené, Alex Vorobiev, Alexander van Delft, Jaime Bernar, Théate Antoine, Nabil Abbar
+// 
+//    This file is part of COMET WEB Community Edition
+//    The COMET WEB Community Edition is the RHEA Web Application implementation of ECSS-E-TM-10-25
+//    Annex A and Annex C.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//        http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// 
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+namespace COMET.Web.Common.Server.Services.NamingConventionService
+{
+    using System.Text.Json;
+
+    using COMET.Web.Common.Services.NamingConventionService;
+
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// The <see cref="NamingConventionService" /> provides static information based on defined naming convention, like for names of
+    /// <see cref="Category" /> to use for example
+    /// </summary>
+    public class NamingConventionService<TEnum> : BaseNamingConventionService<TEnum> where TEnum : Enum
+    {
+        /// <summary>
+        /// The <see cref="ILogger{TCategoryName}" />
+        /// </summary>
+        private readonly ILogger<INamingConventionService<TEnum>> logger;
```

Comment:
having this logger seams to be useless since the base class requires to have an ILogger also.